OnlineFlightSearchAPITestCases/ServiceTests/FlightServicesTests/FlightServiceTests.cs

```diff
@@ -1,36 +1,48 @@
 using Microsoft.Extensions.DependencyInjection;
+using Moq;
 using OnlineFlightSearchAPI.FlightServices;
 using OnlineFlightSearchAPI.Models;
 using OnlineFlightSearchAPI.Repositories;
 using OnlineFlightSearchAPI.Repositories.FlightRepository;
+using OnlineFlightSearchAPI.UnitTests.Mocks;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.Linq;
 using Xunit;
 
 namespace OnlineFlightSearchAPITestCases
 {
-    public class SearchFlightTests
+    public class FlightServiceTests
     {
-        private readonly ISearchFlightService searchFlightService;
+        private ISearchFlightService searchFlightService;
 
-        public SearchFlightTests()
+        private readonly IAirportServices airportService;
+
+        private readonly IAirportRepository airportRepository;
+
+        private FlightRepositoryMock flightRepositoryMocking()
```

Comment:
method name could be better?

Reply:
Removed Separate Mocking in Methods. Moved the mocking to method under test